src/Controls/src/Core/Platform/Windows/CollectionView/ScrollHelpers.cs

```diff
@@ -286,13 +309,26 @@ static async Task<bool> ScrollToItemAsync(ListViewBase list, object targetItem,
 
 			if (targetContainer != null)
 			{
-				await ScrollToTargetContainerAsync(targetContainer, scrollViewer, scrollToPosition);
+				double height = 0;
+
+				if (list.IsGrouping)
+				{
+					height = GetHeaderHeight(list, targetContainer);
+				}
+
+				await ScrollToTargetContainerAsync(targetContainer, scrollViewer, scrollToPosition, height);
 				return true;
 			}
 
 			return false;
 		}
 
+		static double GetHeaderHeight(ListViewBase list, UIElement targetContainer)
+		{
+			var groupHeader = list.GroupHeaderContainerFromItemContainer(targetContainer);
+			return (groupHeader as ListViewHeaderItem).ActualHeight;
```

Comment:
Maybe, before this line check:

`if (groupHeader is ListViewHeaderItem headerItem)`

in that case return the ActualHeight, and in other 0.